examples/sklearnex/pca_spmd.py

```diff
@@ -0,0 +1,40 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy as np
+from mpi4py import MPI
+import dpctl
+from sklearnex.spmd.decomposition import PCA
+
+
+def get_data(data_seed):
+    ns, nf = 15, 21
+    drng = np.random.default_rng(data_seed)
+    X = drng.random(size=(ns, nf))
```

Comment:
Random is mostly the same as uniform - doesn't make sense for PCA.
@KulikovNikita, do you think this number of samples is enough for example?